crates/gpui/src/platform/linux/wayland/client.rs

```diff
@@ -163,6 +163,12 @@ impl WaylandClientStatePtr {
                 state.mouse_focused_window = Some(window);
             }
         }
+        if state.windows.is_empty() {
+            // Drop the clipboard to prevent a seg fault after we've closed all Wayland connections.
+            state.clipboard = None;
```

Comment:
This should also be done when quitting the application normally, e.g. on `crates/gpui/src/platform/linux/platform.rs:131`

Reply:
Actually, there's more problems than that, as we don't drop any windows either. Let's fix all of that in a follow up.